test/cypress/integration/dashboard/dashboard-links.cy.test.js

```diff
@@ -67,27 +65,43 @@ describe('the links are displayed correctly on the dashboard', () => {
       cy.get('#test-payment-link-link').should('exist')
       cy.get('#test-payment-link-link').should('have.class', 'flex-grid--column-half')
 
-      cy.get('#manage-service-link').should('exist')
-      cy.get('#manage-service-link').should('have.class', 'flex-grid--column-half')
-
       cy.get('#request-to-go-live-link').should('exist')
       cy.get('#request-to-go-live-link').should('have.class', 'flex-grid--column-half')
+
+      cy.get('#request-stripe-test-account').should('exist')
+      cy.get('#request-stripe-test-account').should('have.class', 'flex-grid--column-half')
     })
 
-    it('should display 3 links for a test non-sandbox account', () => {
-      cy.task('setupStubs', getStubsForDashboard(gatewayAccountId, 'test', 'worldpay'))
+    it('should display 2 links for a test non-sandbox account', () => {
+      cy.task('setupStubs', getStubsForDashboard(gatewayAccountId, 'test', 'worldpay', 'NOT_STARTED'))
 
       cy.visit(dashboardUrl)
-      cy.get('.links__box').should('have.length', 3)
+      cy.get('.links__box').should('have.length', 2)
 
       cy.get('#payment-links-link').should('exist')
-      cy.get('#payment-links-link').should('have.class', 'flex-grid--column-third')
+      cy.get('#payment-links-link').should('have.class', 'flex-grid--column-half')
 
-      cy.get('#manage-service-link').should('exist')
-      cy.get('#manage-service-link').should('have.class', 'flex-grid--column-third')
+      cy.get('#request-to-go-live-link').should('exist')
+      cy.get('#request-to-go-live-link').should('have.class', 'flex-grid--column-half')
+    })
+
+    it('should display `Stripe test account requested` section if request has been submitted', () => {
```

Comment:
It might be good to check on the content of the link in this test rather than just ids?

Reply:
We don't usually test content in cypress test. We could add templates test to test this ? I will raise separate PR once content is finalised

Reply:
We don't test full content, but we tend to do assertions on elements/text rather than IDs usually as I reckon it gives more confidence that we're displaying the right thing.

Reply:
Usually we also don't use IDs in templates if they're just for testing, we only use them if they're required for application code. We should be able to test most things we used to test in template tests in the Cypress tests as they give us a higher degree of confidence that the template and the server code work together.